Sources/NIO/Utilities.swift

```diff
@@ -17,6 +17,8 @@
 ///
 /// This is currently the only way to do this in Swift: see
 /// https://forums.swift.org/t/support-debug-only-code/11037 for a discussion.
+import CNIOLinux
```

Comment:
why we need the import ?

Reply:
Because of ‘freeifaddrs’, it’s defined for Android in CNIOLinux module(ifaddrs-android.h)